README.md

```diff
@@ -45,5 +45,6 @@ sudo pip3 install -U -r requirements.txt
 
 #### Author
 [Akhilesh Kumar](https://github.com/akhilesh-k), Nov, 2017
+[Shikhar Srivastava](https://github.com/cyber-demon), Dec, 2017
```

Comment:
You should not contribute to repository just by adding name,
Names are added automatically once your PR is merged. Please add some algo or implementation.